src/learning-material/models/LearningMaterial.ts

```diff
@@ -26,11 +26,11 @@ export class LearningMaterial {
   @ApiProperty()
   shortDescription?: string;
 
-  @ApiProperty({
-    description: 'Array of paragraphs that make up the learning outcomes',
-    type: [ParagraphContent],
-  })
-  outcomes: ParagraphContent[];
+  // @ApiProperty({
+  //   description: 'Array of paragraphs that make up the learning outcomes',
+  //   type: [ParagraphContent],
+  // })
+  // outcomes: ParagraphContent[];
```

Comment:
Might be better to remove these lines altogether? You'll be able to access them from Git if you need to in the future.